clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -491,6 +491,31 @@ void UnwrappedLineParser::calculateBraceTypes(bool ExpectClassBody) {
   SmallVector<StackEntry, 8> LBraceStack;
   assert(Tok->is(tok::l_brace));
 
+  constexpr int MaxLookBack = 64;
+  const auto IsAddressOfParenExpression = [](const FormatToken *RightParen) {
+    if (!RightParen || RightParen->isNot(tok::r_paren))
```

Comment:
You already checked `PrevToken` before calling this. No need for a double check.